main.md

```diff
@@ -397,21 +398,19 @@ HTTP request (see also (#request_integrity)).
 
 ## Checking DPoP Proofs {#checking}
 
-To check if a string that was received as part of an HTTP Request is a
-valid DPoP proof, the receiving server MUST ensure that
+To validate a DPoP proof, the receiving server MUST ensure that
 
- 1. that there is not more than one `DPoP` header in the request,
- 1. the string value of the header field is a well-formed JWT,
+ 1. that there is not more than one `DPoP` HTTP request header field,
+ 1. the header field value is a well-formed JWT,
  1. all required claims per (#DPoP-Proof-Syntax) are contained in the JWT,
- 1. the `typ` field in the header has the value `dpop+jwt`,
- 1. the algorithm in the header of the JWT indicates an asymmetric digital
+ 1. the `typ` JOSE header parameter has the value `dpop+jwt`,
+ 1. the `alg` JOSE header parameter indicates an asymmetric digital
     signature algorithm, is not `none`, is supported by the
     application, and is deemed secure,
  1. the JWT signature verifies with the public key contained in the `jwk`
-    header of the JWT,
- 1. the `jwk` header of the JWT does not contain a private key,
- 1. the `htm` claim matches the HTTP method value of the HTTP
-    request in which the JWT was received,
+    JOSE header parameter,
+ 1. the `jwk` JOSE header parameter does not contain a private key,
+ 1. the `htm` claim matches the HTTP method of the current request,
  1. the `htu` claim matches the HTTPS URI value for the HTTP
     request in which the JWT was received, ignoring any query and
     fragment parts,
```

Comment:
```suggestion
 1. the `htu` claim matches the https URI of the HTTP
    request in which the JWT was received, ignoring any query and
    fragment parts,
```

iirc the protocol is `HTTP`. The URI scheme is `https`. 

Probably a more precise statement would be

> matches the target URI [@RFC7231, section 2] of the current request
> ignoring any query and fragment parts, and with the https URI scheme

Consider that the latest HTTP semantics disentangles the request URI from 
HTTP messaging (see https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#rfc.section.7.1.p.2)